cmake/tribits/ci_support/cdash_build_testing_date.py

```diff
@@ -74,8 +74,8 @@ def injectCmndLineOptionsInParser(clp, gitoliteRootDefault=""):
   
   clp.add_option(
     "--cdash-project-start-time", dest="cdashProjectStartTimeUtcStr", type="string", default="",
-    help="Starting time for the CDash testing day in 'HH:MM' in UTC."\
-      + " Check the CDash project settings for the testing date." )
+    help="Starting time for the CDash testing day in 'hh:mmm' in UTC."\
```

Comment:
'hh:mmm' -> 'hh:mm'

Reply:
@rppawlo, thanks for the review!  I just fixed this in the TriBITS 'mastesr' commit https://github.com/TriBITSPub/TriBITS/commit/aaa30da4526b07948d6efef987e676aa8cf85787.  The next time I snapshot TriBITS into Trilinos it will get updated there there.